Plugins/SharedPackagePluginExtensions/Process+TerminationHandler.swift

```diff
@@ -0,0 +1,14 @@
+import Foundation
+
+extension Process {
+  static func HandleErrorTermination(_ process: Process) {
+    if process.terminationStatus != 0 {
+      print("""
+      Hint: You may need to use the --disable-sandbox or --allow-writing-to-package-directory flag
```

Comment:
It wouldn't be `--allow-write-to-package-directory` it would be `-allow-writing-to-directory ${OUTPUT_DIRECTORY}`